website/docs/addon_resolve_admin.md

```diff
@@ -15,96 +15,51 @@ import Tabs from '@theme/Tabs';
 import TabItem from '@theme/TabItem';
 
 ## Resolve requirements
-Due to the way resolve handles python and python scripts there are a few steps required steps needed to be done on any machine that will be using AYON with resolve.
+Resolve is using its own python interpreter which cannot be connected to Ayon. This interpreter needs to have installed PySyde2 and OpenTimelineIO dependencies for Ayon to work correctly. Bellow you can find instructions on how to install them into an installed Python of your choice.
 
-### Installing Resolve's own python 3.6 interpreter.
-Resolve uses a hardcoded method to look for the python executable path. All of tho following paths are defined automatically by Python msi installer. We are using Python 3.6.2.
+Resolve 17 is using Python 3.6.2
+Resolve 18 is using Python 3.9.x
```

Comment:
I believe similar to Fusion the Python 3 of Resolve 18 can use anything between 3.4-3.10 or something. Not sure exactly which versions but as far I know those versions should all work with OpenPype. However, I think it should be clear that here we're just recommending using the same version as OpenPype/AYON if possible to avoid mismatches but should be clear that Resolve itself support different versions.

I believe actually Fusion 17 (and thus Resolve 17) can also already use Python 3.9? Or was that UP TO 3.6?

Reply:
Here is a discussion of that note on BML forum https://forum.blackmagicdesign.com/viewtopic.php?f=21&t=158707

Reply:
But yes I would want to be more clear on that one but it is not very clear what is supported and what verion is change happening. I remember someone mentioned that 3.10 is not working yet but it was probably in early 18 version.

Reply:
Maybe the wording is just more sensible here to state what we recommend. I'm quite sure they support multiple versions in Resolve - which the topic you linked confirms as well.